lib/ts/ink_align.h

```diff
@@ -53,6 +53,12 @@ union Alias64 {
 /** Default alignment */
 #define INK_ALIGN_DEFAULT(size) INK_ALIGN(size, INK_MIN_ALIGN)
 
+static inline size_t
+aligned_spacing(size_t len, size_t block = INK_MIN_ALIGN)
+{
+  return INK_ALIGN(len, block);
+}
```

Comment:
I'm not sure I understand what you mean by "the semantic burden".

You also say, "It does no aligning of any pointer." `aligned_spacing` does nothing but call `INK_ALIGN`, so if `INK_ALIGN` does no aligning, then `aligned_spacing` does no aligning either.

`INK_ALIGN` absolutely rounds a pointer (or whatever value) up to the next multiple of the given power of 2. That sounds very much like something I'd use the verb "align" to describe. I'm not clear what you're claiming here?